packages/server/test/unit/config_spec.js

```diff
@@ -1309,6 +1309,31 @@ describe('lib/config', () => {
         },
       })
     })
+
+    it('resolves a single object', () => {
```

Comment:
We like to link to the original issues when adding a test so we have reference later why we needed this. 

```suggestion
    // https://github.com/cypress-io/cypress/issues/7959
    it('resolves a single object', () => {
```